exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -137,6 +141,8 @@ private void createIndexTemplates() {
         createValueIndexTemplate(ValueType.WORKFLOW_INSTANCE_SUBSCRIPTION);
       }
     }
+
+    indexTemplatesCreated = true;
```

Comment:
What happens if the exporter dies on creating `createValueIndexTemplate` for what ever reason and retry it starts again with createRootIndexTemplate etc. Will this break?

Reply:
No, creating index templates is an upsert operation, so at worse it's doing unnecessary work. This is already the current behavior, in the sense that every time you start the broker we create the templates anyway.

Reply:
Ok but the data get not lost after recreating?

Reply:
No, it's a template, it's not the index itself; it just describes how to index records that match the template.

Reply:
kk thanks